pom.xml

```diff
@@ -91,6 +91,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <beam-top>.</beam-top>
```

Comment:
per discussion, permission granted to add an acceptably-licensed plugin that makes this unnecessary.